arch/lkl/kernel/proc.c

```diff
@@ -46,7 +32,7 @@ int __cpuinfo_accessed = 0;
 static void *c_start(struct seq_file *m, loff_t *pos)
 {
   // TODO: missing correct implementation
```

Comment:
We should probably get the cpuinfo string here and then just copy from it in the read function.  Store it in `m->private`.